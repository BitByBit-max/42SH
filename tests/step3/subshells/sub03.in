(echo "subshell"; exit 42; echo never executed)
echo "subshell exited with status $?"
