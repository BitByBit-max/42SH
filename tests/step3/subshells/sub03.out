subshell
subshell exited with status 42
